components/dashboards-web-component/src/viewer/components/DashboardRenderer.jsx

```diff
@@ -135,7 +135,7 @@ export default class DashboardRenderer extends Component {
 
         GoldenLayoutContentUtils.traverseWidgetContents(goldenLayoutContents, (widgetContent) => {
             let isHeaderShown = _.get(widgetContent, 'props.configs.options.header');
-            if (typeof isHeaderShown != 'undefined'){
+            if (typeof (isHeaderShown) != 'undefined'){
```

Comment:
Here you intention is to use the `isHeaderShown` value if it is not `undefined` or `null`, right? If so, then lets check as `isHeaderShown == null`